### PR TITLE
Random bedsheet spawner.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Spawns a random bedsheet from a list. Mapped it into dorms as well. Also minor fix by removing the ton of lights in that one substation room.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Misc/bedsheets.yml [hunk 1]
```diff
@@ -28,23 +28,18 @@
...
-    QuickEquip: true
-    Slots:
-    - neck
     sprite: Clothing/Neck/Bedsheets/black.rsi
 
 - type: entity
   id: BedsheetBlue
   parent: BedsheetBase
   name: blue bedsheet
+  description:
```
Comment (ZeWaka) [inline Resources/Prototypes/Entities/Objects/Misc/bedsheets.yml:37]: Why would you add a description with no entry?

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Objects/Misc/bedsheets.yml]: Typo

**Review by ShadowCommander — CHANGES_REQUESTED** [review]
Capitalize proper nouns and acronyms. Otherwise, everything looks good.

**Comment by SweptWasTaken** [comment]
> 
> 
> Capitalize proper nouns and acronyms. Otherwise, everything looks good.

Done

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**